src/frontend/src/pages/DesignReviewView.tsx

```diff
@@ -0,0 +1,186 @@
+import { Box, Grid, Typography } from '@mui/material';
+import { DRCViewProps, getBackgroundColor, times, daysOfWeek, TimeSlot } from './DesignReviewCommon';
+import { User } from 'shared';
+import { useState } from 'react';
+import WarningIcon from '@mui/icons-material/Warning';
+import NERFailButton from '../components/NERFailButton';
+import NERSuccessButton from '../components/NERSuccessButton';
+
+const DRCView: React.FC<DRCViewProps> = ({ usersToAvailabilities, iconData }) => {
+  const totalUsers = usersToAvailabilities.size;
+  const availableUsers = new Map<number, User[]>();
+  const unavailableUsers = new Map<number, User[]>();
+  const numberOfTimeSlots = times.length * daysOfWeek.length;
+  const [currentAvailableUsers, setCurrentAvailableUsers] = useState<User[]>([]);
+  const [currentUnavailableUsers, setCurrentUnavailableUsers] = useState<User[]>([]);
+
+  function createAvailableUsers() {
+    for (let time = 0; time < numberOfTimeSlots; time++) {
+      availableUsers.set(time, []);
+    }
+
+    usersToAvailabilities.forEach((availableTimes, user) => {
+      availableTimes.forEach((time) => {
+        const usersAtTime = availableUsers.get(time) || [];
+        usersAtTime.push(user);
+        availableUsers.set(time, usersAtTime);
+      });
+    });
+    return availableUsers;
+  }
+
+  function createUnavailableUsers() {
+    const allUsers = [...usersToAvailabilities.keys()];
+    for (let time = 0; time < numberOfTimeSlots; time++) {
+      const currentUsers = availableUsers.get(time) || [];
+      const currentUnavailableUsers = allUsers.filter((user) => !currentUsers.includes(user));
+      unavailableUsers.set(time, currentUnavailableUsers);
+    }
+    return unavailableUsers;
+  }
+
+  const handleOnMouseOver = (index: number) => {
+    setCurrentAvailableUsers(availableUsers.get(index) || []);
+    setCurrentUnavailableUsers(unavailableUsers.get(index) || []);
+  };
+
+  const handleOnMouseLeave = () => {
+    setCurrentAvailableUsers([]);
+    setCurrentUnavailableUsers([]);
+  };
+
+  const renderDayHeaders = () => {
+    return [
+      <TimeSlot backgroundColor="#D9D9D9" isModal={false} />,
```

Comment:
this backgroundColor should be in the utils file with an appropriate name (in all caps bc it's a constant)